src/utils/dates.js

```diff
@@ -145,12 +153,18 @@ let dates = {
     return dates.startOf(new Date(), 'day')
   },
 
-  yesterday() {
-    return dates.add(dates.startOf(new Date(), 'day'), -1, 'day')
+  tomorrow(date) {
```

Comment:
how about `dayAfter` for a name. tomorrow to me always means the day after today :P

Reply:
Done